sig-k8s-infra/charter.md

```diff
@@ -0,0 +1,186 @@
+escalation point of last resort for more tightly scoped subprojects that
+live within this repo.
+
+#### Cross-cutting and Externally Facing Processes
+
+We prefer (where possible) to delegate ownership, operation and policy
+definition to SIGs that are more directly responsible for a given area
+of the project. However, we reserve the right to halt infrastructure or
+roll back changes if the project as a whole is being negatively impacted.
+
+Some examples for illustrative purposes
+
+##### Access Policies
+
+- We are responsible for ensuring the appropriate members of a SIG have
+  sufficient permissions to troubleshoot and manage their app or
+  infrastructure.
+- However, we will NOT grant overly broad permissions to an overly broad
+  group of people. We will collaborate with SIGs to ensure access is
+  appropriately scoped.
+- We MAY revoke access in the event of a security-related incident
+
+e.g. SIG Release is responsible for who gets what level of access to
+infrastructure used by the release-engineering subproject to cut a kubernetes
+release
+
+##### Artifact Hosting
+
+- We are not responsible for promoting into production artifacts that belong
+  to subprojects owned by other SIGs.
+- However, we MAY revert changes that prevent artifact promotion from
+  functioning.
+
+e.g. SIG Storage is responsible for declaring which CSI-related images should
+be promoted to production, SIG Release is responsible for ensuring those
+images make it to production, and SIG K8s Infra is responsible for ensuring
+that production exists in the first place
+
+##### Community Infra Cluster
+
+- We are responsible for ensuring a community-owned GKE cluster is available
+  to run apps owned by other SIGs.
+- However, we are NOT responsible for ensuring proper functionality of those
+  apps. That is left to the SIGs.
+
+e.g. SIG Scalability is responsible for ensuring perfdash.k8s.io displays
+valid data
+
+##### Project Infrastructure Budget
+
+- We are responsible for enforcing policy on what is considered in-scope and
+  out-of-scope for project infrastructure (and thus, where we spend our
+  infrastructure budget)
+- Crafting such policy is done in collaboration with the Steering Committee
+  (owns project spending) and SIG Architecture (owns kubernetes definition)
+- We MAY delete or scope down infrastructure in the event of unexpected or
+  undue spend
+
+e.g. SIG K8s Infra will deny requests to host artifacts for projects that are
+formerly part of or adjacent to the kubernetes project (e.g. helm, cri-o)
+
+##### Public Names
+
+- We are responsible for enforcing policy on what is considered appropriate
+  or inappropriate for the names of public-facing entities such as DNS
+  records and Google Group names
+- Crafting such policy is done in collaboration with the Steering Committee,
+  SIG Architecture, and SIG Contributor Experience
+
+e.g. Group names that are used to community upon behalf of the project are
```

Comment:
not sure how to parse `Group names that are used to community upon behalf of the project`, rephrase?

Reply:
s/community/communicate, e.g. `contributors@kubernetes.io`

Reply:
Which is basically what I added here: https://github.com/kubernetes/community/pull/5928/files#diff-b55cad1162db37049c36ff01b2892c020ecd9b8ce4a89284850d9bc2b6a6a25aR147-R149